packages/mco/components/protected-applications/protected-apps-list-page.spec.tsx

```diff
@@ -0,0 +1,377 @@
+import * as React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { useScheduler } from '../../../shared/src/hooks';
+import { DRPolicyModel, DRPlacementControlModel } from '../../models/ramen';
+import {
+  EmptyRowMessage,
+  NoDataMessage,
+  EXPANDABLE_COMPONENT_TYPE,
+} from './components';
+import { ProtectedApplicationsListPage } from './list-page';
+
+const unableToFindError = 'Unable to find an element';
+const failingDRPCName = 'test-drpc-1';
+const relocatedDRPCName = 'test-drpc-2';
+const drPolicyName = 'test-policy';
+const deploymentClusterName = 'local-test-cluster';
+const namespaces = ['ns-1', 'ns-2', 'ns-3', 'ns-4'];
+
+let noData = false;
+let noFilteredData = false;
+let filterDRPC = '';
+
+const resetGlobals = () => {
+  noData = false;
+  noFilteredData = false;
+  filterDRPC = '';
+};
+
+const failingDRPC = {
+  apiVersion: 'ramendr.openshift.io/v1alpha1',
+  kind: 'DRPlacementControl',
+  metadata: {
+    name: failingDRPCName,
+    namespace: 'test',
+    annotations: {
+      'drplacementcontrol.ramendr.openshift.io/last-app-deployment-cluster':
+        deploymentClusterName,
+    },
+  },
+  spec: {
+    action: 'Failover',
+    drPolicyRef: { name: 'test-policy' },
+    failoverCluster: 'test-cluster-1',
+    preferredCluster: 'test-cluster-2',
+    placementRef: { name: 'test-ref' },
+    pvcSelector: {},
+    enrolledNamespaces: namespaces,
+    kubeObjectProtection: { captureInterval: '5m' },
+  },
+  status: {
+    lastGroupSyncTime: '2024-03-04T11:38:44Z',
+    lastKubeObjectSyncTime: '2024-03-04T11:38:44Z',
+    phase: 'FailingOver',
+  },
+};
+
+const relocatedDRPC = {
+  apiVersion: 'ramendr.openshift.io/v1alpha1',
+  kind: 'DRPlacementControl',
+  metadata: {
+    name: relocatedDRPCName,
+    namespace: 'test',
+    annotations: {
+      'drplacementcontrol.ramendr.openshift.io/last-app-deployment-cluster':
+        deploymentClusterName,
+    },
+  },
+  spec: {
+    action: 'Failover',
```

Comment:
```suggestion
    action: 'Relocate',
```
no use with this change, just matching phase and action